README.rst

```diff
@@ -19,9 +19,12 @@ Add "favicon" to your INSTALLED_APPS in settings.py::
 
 Add favicon URL patterns to urls.py::
 
+      # from django.conf.urls import url, include
```

Comment:
Why comment instead of removing?

Reply:
Just to let those who use the url function (for example for Django 1.11) to also see how to do.